playbooks/roles/configure/tasks/upload_client_certificates.yml

```diff
@@ -0,0 +1,52 @@
+---
+# DESC
+
+- name: Check if work_dir_remote has a value
+  fail:
+    msg: work_dir_remote is required
+  when: work_dir_remote is not defined or work_dir_remote is none or work_dir_remote == ''
+
+- name: Transfer Certificate Authority to zOS
+  block:
+    - name: Copy APIML_External_Certificate_Authority.cer to local tmp folder
+      copy:
+        src: APIML_External_Certificate_Authority.cer
+        dest: "{{ work_dir_local }}/{{ inventory_hostname }}/"
+      delegate_to: localhost
+
+    - import_role:
+        name: common
+        tasks_from: upload_file
+      vars:
+        filename_to_upload: APIML_External_Certificate_Authority.cer
```

Comment:
Would like to convert the client certificate file paths/names to variables so the user who consumes the playbook can use them to configure their own client certificates, not the one prepared by us.

Similar this also applies to USER-cert.cer.

Reply:
Once we defined these new variables, we can update `playbooks/README.md` `Other Build Variables` section to instruct the users how to update them.